modules/i3s/src/lib/parsers/parse-i3s-tile-content.js

```diff
@@ -38,29 +38,53 @@ function parseI3SNodeGeometry(arrayBuffer, tile = {}) {
   }
 
   const content = tile.content;
-  const {vertexAttributes, attributesOrder} = content.featureData;
+  const {
+    vertexAttributes,
+    attributesOrder,
+    featureAttributes,
+    featureAttributeOrder
+  } = content.featureData;
   // First 8 bytes reserved for header (vertexCount and featureCount)
-  const {vertexCount, byteOffset} = parseHeaders(content, arrayBuffer);
-  const {attributes} = normalizeAttributes(
+  const {vertexCount, byteOffset, featureCount} = parseHeaders(content, arrayBuffer);
+  const {attributes: normalizedVertexAttributes, byteOffset: offset} = normalizeAttributes(
     arrayBuffer,
     byteOffset,
     vertexAttributes,
     vertexCount,
     attributesOrder
   );
 
+  const {attributes: normalizedFeatureAttributes} = normalizeAttributes(
+    arrayBuffer,
+    offset,
+    featureAttributes,
+    featureCount,
+    featureAttributeOrder
+  );
+
+  const attributes = {...normalizedVertexAttributes, ...normalizedFeatureAttributes};
+
   const {enuMatrix, cartographicOrigin, cartesianOrigin} = parsePositions(
+    // @ts-ignore
     attributes.position,
     tile
   );
 
   const matrix = new Matrix4().multiplyRight(enuMatrix);
 
   content.attributes = {
+    // @ts-ignore
```

Comment:
Possible to add types to the functions that generate the `attributes` object so that we can remove the ts-ignores here?

Reply:
Moved code related to attributes in separate function.